backend/tests/integration-async/records/api/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
```

Comment:
do we really need these files in tests?

Reply:
I was getting ```import mismatch``` error otherwise because test files had the same name